Infrastructure.Data/HungerLevels.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Infrastructure.Data
+{
+    public class HungerLevels
+    {
+        public static readonly List<(string, float)> HungerLevelsList = new List<(string, float)>{ ("dead", -0.0f),  
```

Comment:
Class better than tuple

Reply:
Maybe, it would be better to make this list private? Because i noticed, that i don't use it outside the class, and i don't really get, why it isn't good to use tuple in this situation

Reply:
Yeah, maybe private will be good here. 

Tuple - it kind of anonym data, which you can understand before you will see `= new ...`

You can set name for string and float, but class still give more intormation (because you can also add kind of documentation to it)